src/components/action-bar/action-bar-item/action-bar-overflow-menu.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, Prop, h, Event, EventEmitter } from '@stencil/core';
+import {
+    MenuItem,
+    LimelMenuCustomEvent,
+    ListSeparator,
+    ActionBarItem,
+    OpenDirection,
+} from '@limetech/lime-elements';
+
+/**
+ * @private
+ */
+@Component({
+    tag: 'limel-action-bar-overflow-menu',
+    shadow: false,
```

Comment:
Setting `shadow` to false, is this intentional ?

Reply:
That's how we did in the Dock. I don't know. But do we have any good reasons why this is a bad idea?

Reply:
I don't see any reason to use shadow dom if we don't have styling. This component is private, so we have nothing hide in shadows.

(I strongly believe shadow dom by default is absurd. Stencil supports scoped css without shadow dom, so if that's the only reason, we don't strictly need it for scoped styles.)

Reply:
In here, when the shadow is false, we can easily share the styles between the overflow menu and the buttons without doing anything. 

But if we make shadow true, then we will have to import the action button styles into the .scss file of the overflow menu component. It makes it a bit messier.

Reply:
> I don't see any reason to use shadow dom if we don't have styling.

Yes I agree, but I noticed it in many of the new components here in this PR it set to false I was just wondering why we set it to false. The question was not specifically to this line.

@Kiarokh The shadow DOM hides and separates the DOM of a component in order to prevent clashing styles or unwanted side effects. We have building so many components (...Grand parent -> Parent -> child ), we want to separate the DOM of a component. For example we might use same class name in the DOM tree which might have a side effect somewhere

> Stencil supports scoped css without shadow dom

Yes it is "an alternative to using the shadow DOM is using scoped components"

Reply:
> (I strongly believe shadow dom by default is absurd. Stencil supports scoped css without shadow dom, so if that's the only reason, we don't strictly need it for scoped styles.)

(To my knowledge) they didn't support scoped CSS when we started, which is why we used shadow-DOM. Since scoped CSS is now supported, switching to that is perfectly fine with me. I assume there would be a performance boost in using less shadow-DOM. We don't need to rewrite anything either, I think? We could just start using scoped CSS with new components instead, right?

Anyway, one prerequisite for using scoped CSS is that we actually learn know how to use it. You seem to already know, so perhaps you would like to teach the rest of us? I'm thinking that one good way could be to select one of our existing components, based on it having a suitable complexity (not too complex, but not so simple as to make the example meaningless), and change it from using shadow-DOM to using scoped CSS, and then walk us through the changes in a show-and-tell?

Reply:
> In here, when the shadow is false, we can easily share the styles between the overflow menu and the buttons without doing anything.
> 
> But if we make shadow true, then we will have to import the action button styles into the .scss file of the overflow menu component. It makes it a bit messier.

Yeah, this isn't really a separate component, it's more of a sub-component that's broken out for the sake of keeping each component simple to grasp, kind of like breaking out a part of a function into a named helper function.

So I think it makes sense to not have this shadowed 👍

Reply:
> prerequisite for using scoped CSS is that we actually learn know how to use it. You seem to already know, so perhaps you would like to teach the rest of us?

I can't say I know that much. But having a session about it would be nice. Then I'll have to go and learn too 😊